bpf/lib/bpf_task.h

```diff
@@ -96,6 +95,12 @@ FUNC_INLINE __u32 get_task_pid_vnr(void)
 	return upid.nr;
 }
 
+FUNC_INLINE __u32 get_task_pid_vnr_curr(void)
+{
+	struct task_struct *task = (struct task_struct *)get_current_task();
+	return get_task_pid_vnr_by_task(task);
```

Comment:
please put extra line after declaration

Reply:
done

Reply:
there needs to be empty line after declrations, so something like:

```
--- a/bpf/lib/bpf_task.h
+++ b/bpf/lib/bpf_task.h
@@ -98,6 +98,7 @@ FUNC_INLINE __u32 get_task_pid_vnr_by_task(struct task_struct *task)
 FUNC_INLINE __u32 get_task_pid_vnr_curr(void)
 {
        struct task_struct *task = (struct task_struct *)get_current_task();
+
        return get_task_pid_vnr_by_task(task);
 }
```

Reply:
done modify

Reply:
ok, also please add the missing changelog, thanks